shared/src/business/entities/cases/Case.addDocketEntry.test.js

```diff
@@ -24,4 +42,29 @@ describe('addDocketEntry', () => {
       userId: 'irsPractitioner',
     });
   });
+
+  it("should assign docket entry index of '0' to the STIN", () => {
```

Comment:
Following with the clarification of the above test:
```suggestion
  it("should assign docket entry index of '0' to the STIN when not being added to the docket record", () => {
```

Reply:
This rename makes sense to me!!